EventsExpress.Core/Services/EventService.cs

```diff
@@ -233,13 +234,13 @@ public async Task<OperationResult> Edit(EventDTO e)
 
         public EventDTO EventById(Guid eventId) =>
             _mapper.Map<EventDTO>(_db.EventRepository
-                .Get("Photo,Owner.Photo,City.Country,Categories.Category,Visitors.User.Photo")
+                .Get("Photo,Owner.Photo,City.Country,Categories.Category,Visitors.User.Photo,Inventories.UnitOfMeasuring")
                 .FirstOrDefault(x => x.Id == eventId));
 
         public IEnumerable<EventDTO> GetAll(EventFilterViewModel model, out int count)
         {
             var events = _db.EventRepository
-                .Get("Photo,Owner.Photo,City.Country,Categories.Category,Visitors");
+                .Get("Photo,Owner.Photo,City.Country,Categories.Category,Visitors,Inventories.UnitOfMeasuring");
```

Comment:
Why do we need to get all inventory for all items? Do we display it on list somewhere?

Reply:
Solved